src/Pkg.jl

```diff
@@ -117,18 +117,34 @@ a package, also inside that package.
 The `preserve` keyword argument allows you to key into a specific tier in the resolve algorithm.
 The following table describes the argument values for `preserve` (in order of strictness):
 
-| Value             | Description                                                                         |
-|:------------------|:------------------------------------------------------------------------------------|
-| `PRESERVE_ALL`    | Preserve the state of all existing dependencies (including recursive dependencies)  |
-| `PRESERVE_DIRECT` | Preserve the state of all existing direct dependencies                              |
-| `PRESERVE_SEMVER` | Preserve semver-compatible versions of direct dependencies                          |
-| `PRESERVE_NONE`   | Do not attempt to preserve any version information                                  |
-| `PRESERVE_TIERED` | Use the tier which will preserve the most version information (this is the default) |
+| Value             | Description                                                                          |
+|:------------------|:-------------------------------------------------------------------------------------|
+| `PRESERVE_MAX`    | Preserve the state of all existing dependencies and only add those already installed |
```

Comment:
Maybe `PRESERVE_INSTALLED`? `PRESERVE_LOCAL`? `PRESERVE_AVAILABLE`? `MAX` seems a bit non-specific.

Reply:
Updated to `PRESERVE_ALL_INSTALLED`